Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8840,7 +8840,7 @@ void WebPageProxy::revokeGeolocationAuthorizationToken(const String& authorizati
     m_geolocationPermissionRequestManager.revokeAuthorizationToken(authorizationToken);
 }
 
-void WebPageProxy::queryPermission(const ClientOrigin& clientOrigin, const PermissionDescriptor& descriptor, CompletionHandler<void(std::optional<PermissionState>, bool shouldCache)>&& completionHandler)
+void WebPageProxy::queryPermission(const ClientOrigin& clientOrigin, const PermissionDescriptor& descriptor, CompletionHandler<void(std::optional<PermissionState>, bool shouldCache)>& completionHandler)
```

Comment:
I did this because WebPermissionControllerProxy takes in the completionHandler as an rvalue reference, so it when it passes it to WebPageProxy, WebPageProxy should take it in as a lvalue reference right?

Reply:
You did this because you made a mistake and forgot a WTFMove(completionHandler), like I commented below.

Reply:
Ah you're right--I see it. Fixed!